internal/controller/util/secrets_util.go

```diff
@@ -442,8 +442,9 @@ func (sutil *SecretsUtil) policyObject(
 	format TargetSecretFormat,
 	veleroNS string,
 ) *runtime.RawExtension {
+	var object *runtime.RawExtension
+
 	s3SecretRef := corev1.SecretReference{Name: secretName, Namespace: secretNS}
-	object := &runtime.RawExtension{}
```

Comment:
Here we replace pointer to object with nil - this makes sense if the intent of the code is to initialize the pointer only in some cases, so a good change.